pkg/util/mountOpts.go

```diff
@@ -100,12 +100,12 @@ func ProcessOptions(options []string, isTmpfs bool, sourcePath string) ([]string
 				return nil, errors.Wrapf(ErrDupeMntOption, "only one of 'rbind' and 'bind' can be used")
 			}
 			foundBind = true
-		case "z", "Z":
+		case "z", "Z", "O":
```

Comment:
We should probably enforce that O is only allowed if it's the only option here?

Reply:
Added a case to handle 'O', but can we leave the `ProcessOptions` as it is without checking the 'O' option, since the error threw earlier when paring the CLI?